eng/pipelines/common/templates/runtimes/wasm-runtime-and-send-to-helix.yml

```diff
@@ -0,0 +1,88 @@
+parameters:
+  buildConfig: ''
+  archType: 'wasm'
+  osGroup: 'Browser'
+  osSubgroup: ''
+  container: ''
+  testGroup: ''
+  crossrootfsDir: ''
+  readyToRun: false
+  liveLibrariesBuildConfig: ''
+  crossgen2: false
+  compositeBuildMode: false
+  helixQueues: ''
+  stagedBuild: false
+  displayNameArgs: ''
+  runInUnloadableContext: false
+  runtimeVariant: ''
+  variables: {}
+  pool: ''
+  runtimeFlavor: 'mono'
+  runtimeFlavorDisplayName: 'Mono'
+  dependsOn: []
+  #arcade-specific parameters
+  condition: always()
+  continueOnError: false
+  displayName: ''
+  timeoutInMinutes: ''
+  enableMicrobuild: ''
+  gatherAssetManifests: false
+
+
+steps:
+  - script: $(Build.SourcesDirectory)/src/coreclr/build-test$(scriptExt) /p:LibrariesConfiguration=${{ parameters.buildConfig }} -skipnative  -skipstressdependencies -excludemonofailures os Browser wasm
+    displayName: Build Tests
+
+  #  Send tests to Helix
+  - template: /eng/pipelines/common/templates/runtimes/send-to-helix-step.yml
+    parameters:
+      displayName: Send tests to Helix
+      ${{ if eq(parameters.buildConfig, 'debug') }}:
+        buildConfig: 'Debug'
```

Comment:
these variables should already be set by xplat-setup

Reply:
They are setup as variables in xplat-setup, but I need them at template expansion time for send-to-helix-step.yml, right?

Reply:
right, but instead of doing this, you can just pass down `$(buildConfigUpper)` variable as a parameter.